ui/src/common/utils.js

```diff
@@ -259,3 +259,7 @@ export function addCommasToNumber(number) {
 export function getRootOfLocationPathname(pathname) {
   return pathname.split('/')[1];
 }
+
+export function ToFixedNumber(numberOrNull) {
```

Comment:
lowercase `t` in the function name.

Reply:
also, it's OK but you don't have to put this necessarily in `utils.js`, you can just put in the only file where it is used.